Src/Base/AMReX_FabArray.H

```diff
@@ -2506,6 +2510,17 @@ FabArray<FAB>::setDomainBndry (value_type val,
     }
 }
 
+template <class FAB>
+template <class F, std::enable_if_t<IsBaseFab<F>::value,int> FOO>
+void
+FabArray<FAB>::capacityOfFabs(LayoutData<std::size_t>& mem) {
+    AMREX_ASSERT(this->boxArray().CellEqual(mem.boxArray()) &&
+                 this->DistributionMap() == mem.DistributionMap());
```

Comment:
```suggestion
    AMREX_ASSERT(amrex::isMFIterSafe(*this, mem));
```
`CellEqual` is more expensive and it returns false for cases that are safe (e.g., coarsened MultiFabs).